build_debian.sh

```diff
@@ -747,8 +747,7 @@ if [[ $TARGET_BOOTLOADER == uboot ]]; then
         sudo LANG=C chroot $FILESYSTEM_ROOT mv /boot/u${INITRD_FILE} /boot/$INITRD_FILE
     elif [[ $CONFIGURED_ARCH == arm64 ]]; then
         sudo cp -v $PLATFORM_DIR/${sonic_asic_platform}-${CONFIGURED_ARCH}/sonic_fit.its $FILESYSTEM_ROOT/boot/
-		# TODO: fix for bookworm, probably needs arm64 patches in sonic-linux-kernel to be updated
-        #sudo LANG=C chroot $FILESYSTEM_ROOT mkimage -f /boot/sonic_fit.its /boot/sonic_${CONFIGURED_ARCH}.fit
+        sudo LANG=C chroot $FILESYSTEM_ROOT mkimage -f /boot/sonic_fit.its /boot/sonic_${CONFIGURED_ARCH}.fit
```

Comment:
This command is failing with `FATAL ERROR: Couldn't open "/usr/lib/linux-image-5.10.0-23-2-arm64/marvell/ac5-98dx35xx-rd.dtb": No such file or directory`. I think some PRs in sonic-linux-kernel need to be updated/enabled first?

Reply:
looks like below line is missed in your last commit of changing the kernel version.  

https://github.com/sonic-net/sonic-buildimage/blob/bookworm/platform/marvell-arm64/sonic_fit.its#L23

Reply:
Ah, would you be able to make that change here?

Also, I see that you have also opened sonic-net/sonic-linux-kernel#352. That would probably fix the dtb file error.

Reply:
@saiarcot895 I was on leave from last 6days hence the delay in response.

I have taken care this  "https://github.com/sonic-net/sonic-buildimage/blob/bookworm/platform/marvell-arm64/sonic_fit.its#L23" change and updated the PR. 

yes, https://github.com/sonic-net/sonic-linux-kernel/pull/352 would take care of dtb file errors.

Reply:
Can you include the sonic-linux-kernel submodule update in this PR?

Reply:
Thanks Saikrishna for looking into this PR. I have updated sonic-linux-kernel submodule to this PR.